doc/book/src/config.md

```diff
@@ -36,15 +30,16 @@ configuration files, but are overridden by command-line options.
 
 The environment variable name is created by converting the configuration value
 to ["screaming snake case"](https://en.wikipedia.org/wiki/Snake_case), and
-prepending a program-specific prefix. Image that we're evaluating configuration
-values for a program name "maelstrom-prog":
+prepending a program-specific prefix. To set configuration values
+values for "maelstrom-broker", for example, you'd prepend `MAELSTROM_BROKER_`,
+to the environment variable name, like so:
 
 Type    | Example
 --------|----------------------
-string  | `MAELSTROM_PROG_FROB_NAME=string`
-number  | `MAELSTROM_PROG_FROB_SIZE=42`
-boolean | `MAELSTROM_PROG_ENABLE_FROBS=true`
-boolean | `MAELSTROM_PROG_ENABLE_FROBS=false`
+string  | `MAELSTROM_BROKER_FROB_NAME=string`
+number  | `MAELSTROM_BROKER_FROB_SIZE=42`
+boolean | `MAELSTROM_BROKER_ENABLE_FROBS=true`
+boolean | `MAELSTROM_BROKER_ENABLE_FROBS=false`
```

Comment:
I see now that this is a little weird because `maelstrom-broker` doesn't have these options...

Reply:
Right.

So maybe I should introduce the whole notion of a hypothetical program and hypothetical configuration values? Or do you think it would be better to use real programs and real configuration values?